examples/platform/silabs/MatterConfig.cpp

```diff
@@ -350,4 +394,20 @@ extern "C" void vApplicationIdleHook(void)
 #endif // SL_CATALOG_SIMPLE_BUTTON_PRESENT
     SiWxPlatformInterface::sl_si91x_uart_power_requirement_handler();
 #endif
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+#if defined(SL_EM4_SLEEP) && (SL_EM4_SLEEP == 1) 
+    if(chip::ICDConfigurationData::GetInstance().GetICDMode() == chip::ICDConfigurationData::ICDMode::LIT)
+    {
+        uint32_t sleepTimeRemaining = 0;
+        sl_sleeptimer_get_remaining_time_of_first_timer( SL_SLEEPTIMER_ANY_FLAG ,&sleepTimeRemaining);
+        uint32_t idleDuration_seconds = static_cast<int32_t>(chip::ICDConfigurationData::GetInstance().GetIdleModeDuration().count());
```

Comment:
I'm not sure I understand this line - `.count()` should returna uint32.
Might just be a typo but i don't think the `static_cast<int32_t>` is necessary.